text/0000-rfc-process-update.md

```diff
@@ -0,0 +1,166 @@
+---
+tags: ember, ember-cli
+---
+
+* Start Date: 2018-02-04
+* RFC PR: https://github.com/emberjs/rfcs/pull/300
+* Ember Issue: (leave this empty)
+
+# RFC Process Update
+
+## Summary
+
+Introduce the idea of Core Champion to the Ember.js RFC process, merge Ember.js and Ember CLI RFCs repositories,
+and add metadata to make cross-project RFCs easier.
+
+## Motivation
+
+Inspired by Rust, Ember implemented a Request For Commits (RFC) process with the intent to involve the community in discussing the future of the framework.
```

Comment:
I'm confused about RFC abbreviation. I've heard at least about three interpretations:
  - request for **change**
  - request for **comments**
  - and now request for **commits**

Does anybody knows for sure what does "C" stand for?

Reply:
It's a typo, should be Request For Comments [[1]](https://github.com/emberjs/rfcs#ember-rfcs) [[2]](https://github.com/rust-lang/rfcs/#rust-rfcs).